### PR TITLE
[DOCS-6188] UA review for Extension Inspector

### PR DESCRIPTION
**Pull request by anxumalo**
The technical review has already been completed in #154 - this just improves the flow of the page

### CODE REVIEW

File: content-services/latest/develop/extension-inspector.md [hunk 1]
```diff
@@ -2,46 +2,50 @@
 title: Extension Inspector
 ---
 
-Alfresco Extension Inspector allows you to analyse Alfresco extensions and to compare them against a particular Alfresco Content Service version. For example, it helps you to understand how customizations and extensions are affected with a newer version of Alfresco, checking for compliance with our best practices and providing recommendations for upgrade impact and safety.
+Alfresco Extension Inspector allows you to analyze Alfresco extensions and to compare them against a particular Alfresco Content Services version. For example, it helps you to understand how customizations and extensions are affected with a newer version of Content Services, checking for compliance with our best practices and providing recommendations for upgrade impact and safety. It works with Alfresco Community Edition and Alfresco Content Services 5.2.7, 6.2.2, and 7.0 releases.
```
Comment (DenisGabriela) [inline content-services/latest/develop/extension-inspector.md:5]: Actually the tool works with (knows about) many other ACS versions, not only those three.
Here is the complete list of bundled ACS inventories https://github.com/Alfresco/alfresco-extension-inspector/tree/master/extension-inspector-analyser/src/main/resources/bundled-inventories.

File: content-services/latest/develop/extension-inspector.md [hunk 2]
```diff
@@ -103,25 +107,25 @@ When you run the Inventory command, the output is a report in json format with t
...
 
-|Option|Description|
-|------|-----------|
-|`--target-version`|An Alfresco version or a range of Alfresco versions|
-|`--target-inventory`|A file path of an existing WAR inventory|
-|`--verbose`|Verbose output|
+| Option | Description |
+| ------ | ----------- |
+| <nobr>--target-version</nobr> | *Required.* A specific Content Services version or range of versions. |
+| <nobr>--target-inventory</nobr> | *Required.* The file path to an existing WAR inventory. |
```
Comment (DenisGabriela) [inline content-services/latest/develop/extension-inspector.md:127]: `--target-version` and `--target-inventory` are **optional** and mutual exclusive.

File: content-services/latest/develop/extension-inspector.md (hunk 1 above)
Comment (DenisGabriela) [reply content-services/latest/develop/extension-inspector.md:5]: You can rephrase above so it is clearer and concise :).

File: content-services/latest/develop/extension-inspector.md (hunk 2 above)
Comment (oliveralfresco2018) [reply content-services/latest/develop/extension-inspector.md:127]: Added the new bits, thanks.

**Review by oliveralfresco2018 — APPROVED** [review]

**Review by oliveralfresco2018 — APPROVED** [review]

**Review by oliveralfresco2018 — APPROVED** [review]

**Review by oliveralfresco2018 — APPROVED** [review]

File: content-services/latest/develop/extension-inspector.md (hunk 1 above)
Comment (oliveralfresco2018) [reply content-services/latest/develop/extension-inspector.md:5]: Trying to resolve this

**Review by oliveralfresco2018 — APPROVED** [review]

File: content-services/latest/develop/extension-inspector.md (hunk 1 above)
Comment (oliveralfresco2018) [reply content-services/latest/develop/extension-inspector.md:5]: Agreed with George/Denis the message should say 5.2 or later.

**Review by DenisGabriela — APPROVED** [review]
Looks good to me 👍

**Pull request merged**